src/Network/WebSockets/TSSocketManager.m

```diff
@@ -130,6 +130,8 @@ - (void)ensureWebsocket
     // Discard the old socket which is already closed or is closing.
     [self closeWebSocket];
 
+    DDLogWarn(@"Creating new websocket");
```

Comment:
I've seen a bit of jankiness around the websocket lifecycle that might just be edge cases in the simulator - or might not.  Since having a valid websocket is critical, I've added extra logging.